CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Version 2.0.0
 
+- [#478]
+  - **Description:** Changed _dev-only to dev-only
+  - **Products impact:** Script Enhancement
+  - **Addresses:** https://github.com/learningequality/kolibri-design-system/issues/474
+  - **Components:** none
+  - **Breaking:** no
+  - **Impacts a11y:** no
+  - **Guidance:** -
+
```

Comment:
Could you also please paste the link to the PR here on line 19, like it's done for other changelog items?

`[#478]: https://github.com/learningequality/kolibri-design-system/pull/478`

This will ensure that [#478] on line 10 will work as intended.

---

Since you volunteered for another issue - when opening a PR next time, I'd recommend not removing the changelog item from a pull request description and fill it in there. Then, we can just paste it to the changelog.md before merging. This should help to keep it in the right format.

We realize that for some types of work, a core team member will need to fill it out. So generally, any time you don't know what to write in the pull request, you can just keep some changelog fields empty, let us know about it and we will take care of it :)

Reply:
Sure! I'll keep these things in mind next time. 
Thank you for your guidance and patience.

Reply:
Sure, no worries! This is a new process to make creation of release notes easier and we're still figuring things out, especially when it comes to contributions. If you had any feedback or ideas on how to make this straightforward, we're happy to hear back from you.